proxy/auth_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) OpenFaaS authors 2017. All rights reserved.
```

Comment:
Can you update this per - https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md#copyright-notice